phantazm-zombies/src/main/java/com/github/phantazmnetwork/zombies/equipment/gun/shoot/fire/HitScanFirer.java

```diff
@@ -24,20 +24,35 @@
 import java.util.function.BiConsumer;
 import java.util.function.Supplier;
 
+/**
+ * A {@link Firer} that fires using "hit scan". Targets will be immediately found and shot.
```

Comment:
"targets will be immediately found and shot" this sounds very aggressive

Reply:
am I wrong though??????????
(idk how to reword lol)

Reply:
no changes necessary, i just felt the need to comment on it